SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
+                                        completionHandler:(void(^_Nonnull)(GADRemoteModel *
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Similarly, should we create NSDictionary parameters within each method and just request the articleId
+    //from the front end?
+    
+}
+
++ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
+    
+    NSMutableArray <GADArticle *> *articles = [[NSMutableArray alloc] init];
+    NSError *error = nil;
+    NSArray *jsonArray = [NSJSONSerialization JSONObjectWithData:json options:kNilOptions error:&error];
+    
+    if (error != nil) {
+        NSLog(@"Error parsing JSON.");
+    }
+    
+    for (NSDictionary *element in jsonArray) {
+        GADArticle *article = [[GADArticle alloc] init];
+        //Map fields of element to fields of article
+        article.datePublished = element[@"datePublished"];
```

Comment:
Hmm... I agree, but how exactly should that work? Just have a series of constant string variables at the top of the file, like below?

static const string DATE_PUBLISHED = @"datePublished";

I feel like the names of the article fields are already self explanatory so keeping them the same name should be fine?